includes/admin/plugins.php

```diff
@@ -398,6 +400,13 @@ function perflab_render_plugin_card( array $plugin_data ): void {
 			esc_html__( 'Visit plugin site', 'default' )
 		);
 	}
+
+	if ( $show_settings_link ) {
+		$plugin_link = perflab_get_plugin_settings_link( $plugin_data['slug'] );
+		if ( ! empty( $plugin_link ) ) {
+			$action_links[] = $plugin_link;
+		}
+	}
```

Comment:
Is the `$show_settings_link` variable needed? Couldn't this code be put right up above where `$show_settings_link = true;` is currently?

Reply:
@westonruter yes the variable is needed as the suggestion to move the Settings links to the last in the list made the "Learn more" link consistent over all the plugin cards.

I set a variable in the if condition checking if the plugin is active, and at the end of the else condition, added the Settings link into the action links array.